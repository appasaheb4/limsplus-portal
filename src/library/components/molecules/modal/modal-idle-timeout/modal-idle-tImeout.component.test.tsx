import React from 'react';
import {render} from '@utils';
import {ModalIdleTimeout} from './modal-idle-timeout.component';

it('render modalIdleTimeout correctly', () => {
  const modalIdleTimeout = render(
    <ModalIdleTimeout show={true} onClick={() => jest.fn()} />,
  );
  expect(modalIdleTimeout).toMatchSnapshot();
});
