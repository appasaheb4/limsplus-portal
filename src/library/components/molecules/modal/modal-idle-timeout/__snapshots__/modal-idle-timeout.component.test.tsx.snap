// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ModalIdleTimeout component render modalIdleTimeout correctly 1`] = `
Object {
  "asFragment": [Function],
  "baseElement": <body>
    <div>
      <div
        class="container"
      >
        <div
          class="justify-center items-center  overflow-x-hidden overflow-y-auto fixed inset-0 z-50 outline-none focus:outline-none"
        >
          <div
            class="relative w-auto my-6 mx-auto max-w-3xl"
          >
            <div
              class="border-0 rounded-lg shadow-lg relative flex flex-col w-full bg-white outline-none focus:outline-none"
            >
              <div
                class="relative p-6 flex-auto"
              >
                <h3
                  class="text-3xl font-semibold"
                />
                <h3
                  class="text-sm font-semibold"
                />
              </div>
              <div
                class="flex items-center justify-end p-6 border-t border-solid border-gray-300 rounded-b"
              >
                <button
                  class="bg-green-500 text-white active:bg-green-600 font-bold uppercase text-sm px-6 py-3 rounded shadow hover:shadow-lg outline-none focus:outline-none mr-1 mb-1"
                  style="transition: all .15s ease;"
                  type="button"
                >
                  Ok
                </button>
              </div>
            </div>
          </div>
        </div>
        <div
          class="opacity-25 fixed inset-0 z-40 bg-black"
        />
      </div>
    </div>
  </body>,
  "container": <div>
    <div
      class="container"
    >
      <div
        class="justify-center items-center  overflow-x-hidden overflow-y-auto fixed inset-0 z-50 outline-none focus:outline-none"
      >
        <div
          class="relative w-auto my-6 mx-auto max-w-3xl"
        >
          <div
            class="border-0 rounded-lg shadow-lg relative flex flex-col w-full bg-white outline-none focus:outline-none"
          >
            <div
              class="relative p-6 flex-auto"
            >
              <h3
                class="text-3xl font-semibold"
              />
              <h3
                class="text-sm font-semibold"
              />
            </div>
            <div
              class="flex items-center justify-end p-6 border-t border-solid border-gray-300 rounded-b"
            >
              <button
                class="bg-green-500 text-white active:bg-green-600 font-bold uppercase text-sm px-6 py-3 rounded shadow hover:shadow-lg outline-none focus:outline-none mr-1 mb-1"
                style="transition: all .15s ease;"
                type="button"
              >
                Ok
              </button>
            </div>
          </div>
        </div>
      </div>
      <div
        class="opacity-25 fixed inset-0 z-40 bg-black"
      />
    </div>
  </div>,
  "debug": [Function],
  "findAllByAltText": [Function],
  "findAllByDisplayValue": [Function],
  "findAllByLabelText": [Function],
  "findAllByPlaceholderText": [Function],
  "findAllByRole": [Function],
  "findAllByTestId": [Function],
  "findAllByText": [Function],
  "findAllByTitle": [Function],
  "findByAltText": [Function],
  "findByDisplayValue": [Function],
  "findByLabelText": [Function],
  "findByPlaceholderText": [Function],
  "findByRole": [Function],
  "findByTestId": [Function],
  "findByText": [Function],
  "findByTitle": [Function],
  "getAllByAltText": [Function],
  "getAllByDisplayValue": [Function],
  "getAllByLabelText": [Function],
  "getAllByPlaceholderText": [Function],
  "getAllByRole": [Function],
  "getAllByTestId": [Function],
  "getAllByText": [Function],
  "getAllByTitle": [Function],
  "getByAltText": [Function],
  "getByDisplayValue": [Function],
  "getByLabelText": [Function],
  "getByPlaceholderText": [Function],
  "getByRole": [Function],
  "getByTestId": [Function],
  "getByText": [Function],
  "getByTitle": [Function],
  "queryAllByAltText": [Function],
  "queryAllByDisplayValue": [Function],
  "queryAllByLabelText": [Function],
  "queryAllByPlaceholderText": [Function],
  "queryAllByRole": [Function],
  "queryAllByTestId": [Function],
  "queryAllByText": [Function],
  "queryAllByTitle": [Function],
  "queryByAltText": [Function],
  "queryByDisplayValue": [Function],
  "queryByLabelText": [Function],
  "queryByPlaceholderText": [Function],
  "queryByRole": [Function],
  "queryByTestId": [Function],
  "queryByText": [Function],
  "queryByTitle": [Function],
  "rerender": [Function],
  "unmount": [Function],
}
`;
